src/instrumentations/index.ts

```diff
@@ -18,8 +18,10 @@ import { Instrumentation } from '@opentelemetry/instrumentation';
 import { Options } from '../options';
 import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
 import { AwsInstrumentation } from '@opentelemetry/instrumentation-aws-sdk';
-import { configureHttpInstrumentation } from './extentions/http';
 import { diag } from '@opentelemetry/api';
+import { configureHttpInstrumentation } from './extentions/http';
+import { configureAWSInstrumentation } from './extentions/aws/aws_sdk';
+import { configureRedisnstrumentation } from './extentions/redis';
 
 export function getInstrumentations(options: Options): Instrumentation[] {
   const instrumentations = getNodeAutoInstrumentations();
```

Comment:
shouldn't we push also redis instrumentation here?

Reply:
The redis instrumentation is in: otel-js/src/instrumentations/extentions/redis.ts - this file creates the "configureRedisnstrumentation" in which we use here in index.ts